src/borrower/feed/navfeed.sol

```diff
@@ -262,9 +275,10 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets, FixedPoint {
         if (debt != 0) {
             fv = calcFutureValue(loan, debt, maturityDate_, recoveryRatePD[risk[nftID_]].value);
             buckets[maturityDate_].value = safeAdd(buckets[maturityDate_].value, fv);
-            futureValue[nftID_] = fv;
         }
 
+        futureValue[nftID_] = fv;
```

Comment:
We need to update it to potentially allow partial borrow, correct?

Reply:
yes, exactly